examples/tests/valid/extern_implementation.test.w

```diff
@@ -1,12 +1,12 @@
 bring cloud;
 
 class Foo {
-  extern "./external_js.js" pub static getGreeting(name: str): str;
-  extern "./external_js.js" static inflight regexInflight(pattern: str, text: str): bool;
-  extern "./external_js.js" static inflight getUuid(): str;
-  extern "./external_js.js" static inflight getData(): str;
-  extern "./external_js.js" pub static inflight print(msg: str): void;
-  extern "./external_js.js" pub static preflightBucket(bucket: cloud.Bucket, id: str): Json;
+  extern "./external_ts.ts" pub static getGreeting(name: str): str;
+  extern "./external_ts.ts" static inflight regexInflight(pattern: str, text: str): bool;
+  extern "./external_ts.ts" static inflight getUuid(): str;
+  extern "./external_ts.ts" static inflight getData(): str;
+  extern "./external_ts.ts" pub static inflight print(msg: str): void;
+  extern "./external_ts.ts" pub static preflightBucket(bucket: cloud.Bucket, id: str): void;
```

Comment:
Is `.js` not supported anymore? (wondering why all of these are changed)

Reply:
It is. I only changed one file, there are other usages of .js extern.

Honestly though if extern supports .ts in both preflight and inflight, I expect most people to default to .ts.